src/client/replication_ddl_client.cpp

```diff
@@ -1580,5 +1580,19 @@ replication_ddl_client::query_bulk_load(const std::string &app_name)
     return call_rpc_sync(query_bulk_load_rpc(std::move(req), RPC_CM_QUERY_BULK_LOAD_STATUS));
 }
 
+error_code replication_ddl_client::detect_hotkey(const dsn::rpc_address &target,
```

Comment:
Does the commend support send multi nodes？if support, you should pass  `targets`

Reply:
I think there is no need to support sending to multi nodes right now.